### ISSUE TITLE
Bump version for Venu3 support

### ISSUE THREAD

**Issue by evilwombat** [tymmej/HikeField#1]
Hello. Is there a chance you could bump the compatibility list to add support for Venu3? Or is that not so simple? Thank you

**Comment by evilwombat** [comment]
Or, would you mind please pushing the latest version to github? It seems the version in the IQ store is somewhat ahead of the code here. Thank you

**Comment by tymmej** [comment]
Hi, I tried to update version using new CIQ. Unfortunately, there were some changes in API and project does not build with newest version.

As I am no longer using it I do not invest time in it.
However, I'll merge any pull request resolving issues.

I've pushed newest version.

**Comment by evilwombat** [comment]
Thank you for pushing the code!

I was able to compile and run on Venu 3. I've never used this language before, but it seems like all I needed to do was remove the 'return' statement from onShow() and it compiles. I still need to learn the SDK.

Layout on Venu 3 is pretty good. I think the time is slightly cut off. I reduced the size of the font and it is better. I may shift it down a little, so there is more room for time and less room for notifications. Maybe will look later. Still need to test properly, but too dark for hiking right now. Thanks again!

**Comment by evilwombat** [comment]
Okay, I think I understand how some of this works. I just refactored some of the layout code to be a little more abstract, and tweaked the layout for Venu 3 (time was getting a little cut off because round screen). It's a good thing for starting to learn the language. THANK YOU again for releasing source code - this is what makes it possible for random people to get started on projects they care about.

**Comment by evilwombat** [comment]
Thanks again for pushing the code. I think I managed to add all the
features I wanted. Would you mind if I push my updated app to the Garmin
store? I would use a new app UUID, name it "HikeField v2", and give you
credit in the description. The app would be free and open source.

Thanks
-Steve

**Comment by tymmej** [comment]
As long as your app stays free I have no problem with it. 

Please provide me link to new app so I’ll update description of HikeField.

I looked if there’s option to transfer ownership of app but unfortunately not :(.

**Comment by evilwombat** [comment]
Thanks! I just finally uploaded the first version to the Garmin store, now
waiting for approval. Took a while to finally do the graphics, and I kept
changing things at the last minute. Tomorrow will do another round of final
testing, while they approve everything else. I'll send you a link when the
app page becomes visible.

I've also pushed code here, if you are interested:
https://github.com/evilwombat/HikeFieldv2
This is my first time writing for Garmin, so it may not be the best
organization.

-Steve
